### DISCUSSION TITLE
How to change size Column in Grid.ts? I Used [Width(50)]in column.cs But Not Change !!!! Why

### DISCUSSION THREAD

**Discussion by ra-bedo11** [serenity-is/Serenity#6170] [Help] [answered]
![Column1](https://user-images.githubusercontent.com/29665710/147857666-07f08f1e-89a7-457b-a45a-8b0aa819db86.jpg)
**This My Code in grid.ts**

namespace Serene3.Default {
    @Serenity.Decorators.registerClass()
    export class ViewSalesDetailsGrid extends _Ext.GridBase<ViewSalesDetailsRow, any> {

        protected getColumnsKey() { return 'Default.ViewSalesDetails'; }
        protected getDialogType() { return ViewSalesDetailsDialog; }
        protected getIdProperty() { return ViewSalesDetailsRow.idProperty; }
        protected getInsertPermission() { return ViewSalesDetailsRow.insertPermission; }
        protected getLocalTextPrefix() { return ViewSalesDetailsRow.localTextPrefix; }
        protected getService() { return ViewSalesDetailsService.baseUrl; }

        constructor(container: JQuery, options) {
            super(container, options);

            //this.slickGrid.setOptions({ defaultColumnWidth:100 })
        }
        protected getButtons(): Serenity.ToolButton[] {
            var btn = super.getButtons();
            btn = [];
            return btn;
        }
    }
}
**this my code in Column.Cs**

namespace Serene3.Default.Columns
{
    [ColumnsScript("Default.ViewSalesDetails")]
    [BasedOnRow(typeof(Entities.ViewSalesDetailsRow), CheckNames = true)]
    public class ViewSalesDetailsColumns
    {
        [EditLink]
        [Width(50)]
        public String ProductProductCode { get; set; }
        [Width(50)]
        public String ProductProductNameAr { get; set; }
        [Width(50)]
        public String UnitUnitNameAr { get; set; }
        [Width(50)]
        public Decimal SalesQte { get; set; }
        [Width(50)]
        public Decimal SalesPrice { get; set; }  
        [Width(50)]
        public Decimal DiscountPrice { get; set; }
        [Width(50)]
        public Decimal Total { get; set; }
        [Width(50)]
        public Decimal VatPrice { get; set; }
        [Width(50)]
        public Decimal TotalBill { get; set; }
    }
}

**Comment by dfaruque** [comment, marked as answer]
Use following
`[Width(50, Min = 50)]` or `[Width(50, Max = 50)`

**Reply by ra-bedo11** [reply]
Its Ok
